### PR TITLE
feat: add mlflow guidance in website and fix some installation pieces

### PR DESCRIPTION
**Pull request by serena-ruan**
# Summary

Add samples of Mlflow usage with SynapseML on the website and fix some installation guidance for the `next` version.

# Tests

Tested website rendering correctly.

### CODE REVIEW

**Comment by serena-ruan** [comment]
FYI: https://github.com/microsoft/SynapseML/issues/1407

File: website/docs/mlflow/examples.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                .setLocation("eastus")
+                .setTextCol("text")
+                .setOutputCol("prediction")
+                .setErrorCol("error")
+                .setLanguageCol("language"))
+
+    display(sentiment.transform(df))
+
+    # Wrap it as a stage in the PipelineModel
+    sentimentM = PipelineModel(stages=[sentiment])
```
Comment (mhamilton723) [inline website/docs/mlflow/examples.md:129]: nit: maybe just call this sentimentModel lol

File: website/docs/mlflow/examples.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            .setRawPredictionCol("rawPrediction")
+            .setDefaultListenPort(12402)
+            .setNumLeaves(5)
+            .setNumIterations(10)
+            .setObjective("binary")
+            .setLabelCol("labels")
+            .setLeafPredictionCol("leafPrediction")
+            .setFeaturesShapCol("featuresShap"))
+
+    lightbmCM = lightgbmClassifier.fit(df_trans)
```
Comment (mhamilton723) [inline website/docs/mlflow/examples.md:80]: maybe just lightgbm_model

File: website/docs/mlflow/examples.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+```python
+import mlflow
+from synapse.ml.featurize import Featurize
+from synapse.ml.lightgbm import *
+from synapse.ml.train import ComputeModelStatistics
+
+with mlflow.start_run():
+
+    featureColumns = ["Number of times pregnant","Plasma glucose concentration a 2 hours in an oral glucose tolerance test",
```
Comment (mhamilton723) [inline website/docs/mlflow/examples.md:33]: nit: snake case in python examples

**Review by mhamilton723 — CHANGES_REQUESTED** [review]
This is an awesome PR!! only a few nits

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**